### PR TITLE
Update k8s.io/docs/concepts/workloads/controllers/job

### LINKED ISSUE #34093: Improvement for k8s.io/docs/concepts/workloads/controllers/job/
Why don't you just provide a completed example of a job with Labels, Parallelism, Completions instead of getting them from describing the job?

### PR DESCRIPTION
**Pull request by mk46**
Fixes #34093
/language en
/kind feature
/cc @sftim

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -49,15 +49,10 @@ The output is similar to this:
 job.batch/pi created
 ```
 
-Check on the status of the Job with `kubectl`:
+Check on the status of the Job with `kubectl` and output is similar to this:
```
Comment (zacharysarah) [inline content/en/docs/concepts/workloads/controllers/job.md:52]: No need to replace, the original text is fine.

**Review by zacharysarah — CHANGES_REQUESTED** [review]
@mk46 Nice implementation! ✨ Only one change required, otherwise LGTM

File: content/en/docs/concepts/workloads/controllers/job.md (hunk 1 above)
Comment (mk46) [reply content/en/docs/concepts/workloads/controllers/job.md]: Reverted. Thanks!!

File: content/en/docs/concepts/workloads/controllers/job.md [hunk 2]
```diff
@@ -51,13 +51,8 @@ job.batch/pi created
...
 Check on the status of the Job with `kubectl`:
 
-```shell
-kubectl describe jobs/pi
-```
-
-The output is similar to this:
-
-```
+{{< tabs name="Check status of Job" >}}
```
Comment (zacharysarah) [inline content/en/docs/concepts/workloads/controllers/job.md:54]: Good call, I think this follows nicely from the previous line.

**Review by zacharysarah — APPROVED** [review]
👍🏻

**Pull request merged**